.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -208,10 +208,15 @@ jobs:
           ref: ${{ github.event.inputs.gitref }}
           token: ${{ secrets.GH_PAT_MAINTENANCE_OSS }} # This token is what allows us to commit directly to master
       - name: Bump CDK dependency of source-declarative-manifest
+        timeout-minutes: 10
         run: |
           cd airbyte-integrations/connectors/source-declarative-manifest
-          # --no-cache is used to ensure the latest version is fetched. Let's see if we need to add a wait...
-          poetry add airbyte-cdk==${{needs.bump-version.outputs.new_cdk_version}} --no-cache
+          while true; do
```

Comment:
This seems very odd. If it fails, will there be a good error message to indicate the failure?

Reply:
If it continues to fail, the github action will timeout. Then we will get the standard slack error message of us not being able to update the source-declarative-manifest version

Are you worried about it failing to actually add the dependency (compared to if it can't resolve it in time)?

Reply:
Ah I see! `manage.sh` took over 5 minutes of processing before getting to the "install the newly published version of the cdk into the docker image" step of the job that happened directly after the cdk publish to pypi step. This gave it time to become available. 

This job by comparison gets to the installation step in 42 seconds after the publish job, which doesn't seem to be quite enough time (sometimes).

Reply:
The output of both the failures and the successes shoudl still print to the GHA logs. I'll add an echo about the retry to make it make more sense